service.subtitles.titlovi-com-official/service.py

```diff
@@ -0,0 +1,592 @@
+# -*- coding: utf-8 -*-
+
+import os
+from sys import path as sys_path, argv as sys_argv
+import re
+import xbmc
+import xbmcaddon
+import xbmcgui
+import xbmcplugin
+import xbmcvfs
+import requests
+import simplecache
+import time
+import unicodedata
+import io
+import codecs
+from datetime import datetime, timedelta
+from urllib.parse import parse_qs
+from urllib.parse import quote_plus
+from zipfile import ZipFile
+
+addon = xbmcaddon.Addon()
+video_info = xbmc.InfoTagVideo()
+
+author = addon.getAddonInfo('author')
+script_id = addon.getAddonInfo('id')
+script_name = addon.getAddonInfo('name')
+version = addon.getAddonInfo('version')
+get_string = addon.getLocalizedString
+
+script_dir = xbmcvfs.translatePath(addon.getAddonInfo('path'))
+profile = xbmcvfs.translatePath(addon.getAddonInfo('profile'))
+libs_dir = xbmcvfs.translatePath(os.path.join(script_dir, 'resources', 'lib'))
+temp_dir = xbmcvfs.translatePath(os.path.join(profile, 'temp', ''))
+
+player = xbmc.Player()
+
+sys_path.append(libs_dir)
```

Comment:
Still here...and afaik it is not even used

Reply:
I have added relative imports as you requested on line 4.

Reply:
Yeah but the sys.path.append has to go away

Reply:
Im using `sys_path`, n ot `sys.path`. That is underscore, not dot. I have imported `path` under alias `sys_path` from `sys`  using relative import. Please check line 4.

Reply:
Nop, I requested to use relative module import as an alternative to sys.path append, not for you to use sys.path as a relative import. As far as I can see you don't even have anything under the lib folder so this can be removed altogether

Reply:
Sorry for misunderstanding. I have now removed appending to sys.path. As you said, it looks like it wasn't event needed.

Reply:
@enen92 Hello, is there any new updates about this commit?